03_simon_says/simon_says.rb

```diff
@@ -1 +1,27 @@
 #write your code here
+def echo(input)
+    "#{input}"
```

Comment:
❕ Ruby can handle returning this value without you interpolating to a string.

```suggestion
   input
```